README.md

```diff
@@ -1,2 +1,70 @@
-# .github
-Organization-wide repository for common governance documents.
+# Notary Project Overview
+
+The Notary Project is a set of specifications and tools intended to provide a cross-industry standard for securing software supply chains through image signing and verification, signature portability, and key management. 
```

Comment:
Note: Just to confirm that if the key management is indeed in the scope of Notary Project. Here the key management refers to key creation, key storage, key distribution, key rotation, key revocation.

Reply:
It is covered by TUF-based notary but not notation.

Reply:
Noted